plugins/dell/fu-plugin-dell.c

```diff
@@ -826,7 +826,10 @@ fu_plugin_update_prepare (FuPlugin *plugin,
                           FuDevice *device,
                           GError **error)
 {
-	return fu_dell_toggle_flash (device, error, TRUE);
+	gboolean result = fu_dell_toggle_flash (device, error, TRUE);
```

Comment:
I'd rather this be:

    if (!fu_dell_toggle_flash (device, error, TRUE))
        return FALSE;
    g_usleep (DELL_FLASH_MODE_DELAY * G_USEC_PER_SEC);
    return TRUE;

... and then we always stick to a "bail on error" return scheme.